usr/src/lib/smbclnt/libfknsmb/Makefile

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
Just a small grip here.  This one file consists of nothing more than a single statement.  The copyright block here is crazy long, and I can't imagine that its enforceable.   Maybe you could have *not* copied the file and started a new one with the shorter CDDL blurb instead?  Ok, I'll get off my soapbox now.

Reply:
Perhaps so.  I just try to be careful about copying and copyrights.

Reply:
I changed out the ones with small "real" content.